crowbar_framework/app/assets/stylesheets/content/panel.scss

```diff
@@ -0,0 +1,55 @@
+.panel-heading {
+  border-bottom: 0;
+}
+
+.panel .panel-heading h3 {
+  padding: 0;
+  margin: 0;
+}
+.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Each selector in a comma sequence should be on its own single line